build.gradle

```diff
@@ -1211,6 +1211,8 @@ project(':core') {
     //By default gradle does not handle test dependencies between the sub-projects
     //This line is to include clients project test jar to dependant-testlibs
     from (project(':clients').testJar ) { "$buildDir/dependant-testlibs" }
+    // added in order to pass the kafka_log4j_appender.py test
```

Comment:
"log4j-appender is not in core dependencies, so we add it to dependant-testlibs to avoid ClassNotFoundException in running  kafka_log4j_appender.py"